static/app/components/replays/table/replayTableHeader.tsx

```diff
@@ -20,18 +20,27 @@ type Props = {
   replays: ReplayListRecord[];
   onSortClick?: (key: string) => void;
   sort?: Sort;
+  stickyHeader?: boolean;
 };
 
-export default function ReplayTableHeader({columns, replays, onSortClick, sort}: Props) {
+export default function ReplayTableHeader({
+  columns,
+  replays,
+  onSortClick,
+  sort,
+  stickyHeader,
+}: Props) {
   const listItemCheckboxState = useListItemCheckboxContext();
   const {countSelected, isAllSelected, isAnySelected, queryKey, selectAll, selectedIds} =
     listItemCheckboxState;
   const queryOptions = parseQueryKey(queryKey).options;
   const queryString = queryOptions?.query?.query;
 
+  const headerStyle = stickyHeader ? ({position: 'sticky', top: 0} as const) : {};
+
   return (
     <Fragment>
-      <TableHeader>
+      <TableHeader style={{...headerStyle}}>
```

Comment:
l: i don't think the spread is necessary here?

```suggestion
      <TableHeader style={headerStyle}>
```